internal/database/sqlcommon/message_sql_test.go

```diff
@@ -49,7 +49,7 @@ func TestUpsertE2EWithDB(t *testing.T) {
 			ID:   msgID,
 			CID:  nil,
 			Type: fftypes.MessageTypeBroadcast,
-			Identity: fftypes.Identity{
+			SignerRef: fftypes.SignerRef{
```

Comment:
Note this is just an internal structure name change for grouping of fields - the JSON field names are still the same (`key` - the signing key, and `author` - the DID)